### ISSUE TITLE
Ubuntu 20.04: Can't click in window

### ISSUE THREAD

**Issue by ekapilik** [HorrorPills/ChatGPT-Gnome-Desktop-Extension#16]
## System Info
```
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.5 LTS
Release:	20.04
Codename:	focal

uname -a
... #63-Ubuntu SMP ... 2022 x86_64 x86_64 x86_64 GNU/Linux
```
## Gnome Extension managed via chrome-gnome-shell
```
apt search chrome-gnome-shell
Sorting... Done
Full Text Search... Done
chrome-gnome-shell/focal,focal,now 10.1-5 all [installed]
  GNOME Shell extensions integration for web browsers
```

## Issue
I can't click anywhere in the window. Tab works but can't get to the "Verify you are human" button.
![chatgpt-cant-click](https://user-images.githubusercontent.com/19395760/213756593-8969f900-ef0e-4856-9228-d22af9f95c9d.png)

**Comment by nocnoc94** [comment]
I am having the same issue on Ubuntu 22.04 LTS GNOME 42.5

**Comment by Redoxahmii** [comment]
same issue

**Comment by FreedomBen** [comment]
Likewise, Fedora 36 Gnome 42.5.  I'm guessing the html/script that is loaded has "anti-bot" detection that makes it un-clickable, but there may be a setting or flag we can flip.

This is surely a reaction to the "excessive demand" the system has been experiencing.  Even through the site it's only available about 1/3 the time.  I doubt they will ever relax this requirement however.

![Screenshot from 2023-01-09 15-26-01](https://user-images.githubusercontent.com/4177139/213787042-fc223728-e8f3-49b2-b79a-5c3121489baf.png)

**Comment by FreedomBen** [comment]
Likewise, Fedora 36 Gnome 42.5. I'm guessing the html/script that is loaded has "anti-bot" detection that makes it un-clickable, but there may be a setting or flag we can flip.

This is surely a reaction to the "excessive demand" the system has been experiencing. Even through the site it's only available about 1/3 the time. I doubt they will ever relax this requirement however.

![2023-01-20-12-14-21_screenshot](https://user-images.githubusercontent.com/4177139/213787244-ee39bc0f-ec46-435c-b418-bdae1ae7d17b.png)

**Comment by HorrorPills** [comment]
Hello, thanks for reporting the issue.

As mentioned on the main page, this is a known issue that is getting looked into. The mouse interaction problem exists no matter what website is loaded. 

To tab to the verification, press tab twice after loading the page, then press enter or space. This should check the verification box.

Any contribution to the project is of course welcome.

**Comment by moh-slimani** [comment]
> Hello, thanks for reporting the issue.
> 
> As mentioned on the main page, this is a known issue that is getting looked into. The mouse interaction problem exists no matter what website is loaded.
> 
> To tab to the verification, press tab twice after loading the page, then press enter or space. This should check the verification box.
> 
> Any contribution to the project is of course welcome.

didn't work

**Comment by faheemKamboh** [comment]
> Hello, thanks for reporting the issue.
> 
> As mentioned on the main page, this is a known issue that is getting looked into. The mouse interaction problem exists no matter what website is loaded.
> 
> To tab to the verification, press tab twice after loading the page, then press enter or space. This should check the verification box.
> 
> Any contribution to the project is of course welcome.

Thanks, it worked for me, 

@moh-slimani ,
Just for information, you need to focus the window before you hit the tab twice ).

**Comment by dortonway** [comment]
I have the same issue with mouse

**Comment by Zatacka** [comment]
Is there going to be a version for Windows?

**Comment by JaimeZX** [comment]
I get a completely blank grey window with no text whatsoever.  (Ubuntu 22.04)

**Comment by kurokirasama** [comment]
same issue, justo commented to get updates in this. Regards.

**Comment by mzsuetam** [comment]
> > Hello, thanks for reporting the issue.
> > As mentioned on the main page, this is a known issue that is getting looked into. The mouse interaction problem exists no matter what website is loaded.
> > To tab to the verification, press tab twice after loading the page, then press enter or space. This should check the verification box.
> > Any contribution to the project is of course welcome.
> 
> didn't work

Didn't work for me for the first time and later for Enter, but finally worked after spamming space

**Comment by lupinx2** [comment]
I've been looking into this the last couple days with very little progress; currently I'm trying to rewrite most of extension.js to use a different gtk container to see if that fixes it.  The odd thing is that if you test the window with a website that reacts to mouse movement, you can see that the webview does register an event when the cursor enters or exits its draw area, it just doesn't track any other movement or mouse down events, or if it does they don't propagate properly even though the widget has focus.

**Comment by HorrorPills** [comment]
> I've been looking into this the last couple days with very little progress; currently I'm trying to rewrite most of extension.js to use a different gtk container to see if that fixes it. The odd thing is that if you test the window with a website that reacts to mouse movement, you can see that the webview does register an event when the cursor enters or exits its draw area, it just doesn't track any other movement or mouse button down events, or if it does they don't propagate properly even though the widget has focus.

Yes, its the hardest nut to crack at the moment. I am battling with it too.
